src/layers.js

```diff
@@ -16,21 +16,30 @@ exports.layerInfo = {
 function getNodeLayerProps(func, service, userLayerVersion) {
     const optsWithCR = {
         layerName: 'thundra-lambda-node-layer',
-        defaultLayerVersion: '43',
+        defaultLayerVersion: '57',
         needHandlerDelegation: false,
         customRuntime: true,
     }
 
     const optsWithoutCR = {
         layerName: 'thundra-lambda-node-layer',
-        defaultLayerVersion: '43',
+        defaultLayerVersion: '57',
         needHandlerDelegation: true,
         thundraHandlerName:
-            '/opt/nodejs/node_modules/@thundra/core/dist/handler.wrapper',
+            'thundra_handler.wrapper',
```

Comment:
This might be problematic for the ones who are using Thundra layer without custom runtime before the version we added  `thundra_handler.wrapper`

Reply:
👍